vortexasdk/endpoints/attributes.py

```diff
@@ -0,0 +1,59 @@
+"""Attributes Endpoint."""
+from typing import List, Union
+from vortexasdk.endpoints.endpoints import ATTRIBUTES_REFERENCE
+from vortexasdk.endpoints.attributes_result import AttributeResult
+from vortexasdk.operations import Reference, Search
+from vortexasdk.utils import convert_to_list
+
+
+class Attributes(Reference, Search):
+    """Attributes endpoint."""
+
+    def __init__(self):
+        Reference.__init__(self, ATTRIBUTES_REFERENCE)
+        Search.__init__(self, ATTRIBUTES_REFERENCE)
+
+    def load_all(self) -> AttributeResult:
+        """Load all attributes."""
+        return self.search()
+
+    def search(
```

Comment:
We're shadowing the inbuilt python keyword `type` here.

I don't think it's too big a problem to do so here, but we can annotate the code to alert any linters / code quality checkers that we're aware of this.

```suggestion
	# noinspection PyShadowingBuiltins
    def search(
```